upgrade_tests/upgrade_through_versions_test.py

```diff
@@ -497,7 +497,7 @@ def upgrade_scenario(self, populate=True, create_schema=True, rolling=False, aft
                 self._check_counters()
                 self._check_select_count()
 
-        if self.cluster.version() >= '5.1':
+        if self.cluster.nodelist()[0].get_cassandra_version() >= '5.1':
```

Comment:
This I don't believe is correct. I can see no valid reason why the cluster would be on 5.1 but node1 would not be & in fact that would constitute a failure at this point in the test.

Reply:
we can do forward upgrade tests.
so the cluster is configured with 5.0, and then one node is upgraded to 5.1

Reply:
my bad – this method upgrades all nodes and the cluster first.